### PR TITLE
fix: simulator call crashes video rendering - WPB-9413

### PR DESCRIPTION
**Pull request by markusfassbender**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9413" title="WPB-9413" target="_blank">BugWPB-9413</a>  [iOS] Simulator call crashes app after other participant enables video
  </td></table>
  <br />
 

### Issue

- Updates AVS to a new version, that uses the newer technology Apple Metal and replaces deprecated OpenGL.
- Fixes a bug where notifications in 1-1 calls have been processed twice. It lead to weird bugs in the rendering, but probably effected other things as well.

### Testing

- See ticket
- Playground build 3.113.0 (11712) is ready to test  🚀

---

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [x] Adds/updates automated tests.

### CODE REVIEW

**Review by johnxnguyen — APPROVED** [review]

**Review by agisilaos — APPROVED** [review]

File: wire-ios-sync-engine/Tests/Source/Synchronization/Strategies/CallingRequestStrategyTests.swift [hunk 1]
```diff
@@ -645,40 +645,6 @@ class CallingRequestStrategyTests: MessagingTest {
...
-            "type": "conversation.otr-message-add"
-        ] as [String: Any]
-
-        let updateEvent = ZMUpdateEvent(fromEventStreamPayload: payload as ZMTransportData, uuid: UUID())!
-
-        sut.callCenter?.isMuted = false
-
-        // WHEN
-        syncMOC.performAndWait {
-            sut.processEventsWhileInBackground([updateEvent])
```
Comment (typfel) [inline wire-ios-sync-engine/Tests/Source/Synchronization/Strategies/CallingRequestStrategyTests.swift:675]: Should we really remove this test? 

Isn't it enough to replace `sut.processEventsWhileInBackground()` with `processEvents()`?

File: wire-ios-sync-engine/Source/Synchronization/Strategies/CallingRequestStrategy.swift [hunk 2]
```diff
@@ -240,11 +240,6 @@ public final class CallingRequestStrategy: AbstractRequestStrategy, ZMSingleRequ
         events.forEach(processEvent)
     }
 
-    public func processEventsWhileInBackground(_ events: [ZMUpdateEvent]) {
```
Comment (typfel) [inline wire-ios-sync-engine/Source/Synchronization/Strategies/CallingRequestStrategy.swift:243]: @netbe I think we should consider cherry-picking this fix to release

**Review by typfel — CHANGES_REQUESTED** [review]

File: wire-ios-sync-engine/Tests/Source/Synchronization/Strategies/CallingRequestStrategyTests.swift (hunk 1 above)
Comment (markusfassbender) [reply wire-ios-sync-engine/Tests/Source/Synchronization/Strategies/CallingRequestStrategyTests.swift:675]: Thanks, I've reverted the change to remove the test and just updated it.

File: wire-ios-sync-engine/Source/Synchronization/Strategies/CallingRequestStrategy.swift (hunk 2 above)
Comment (markusfassbender) [reply wire-ios-sync-engine/Source/Synchronization/Strategies/CallingRequestStrategy.swift:243]: @typfel @netbe I opened a PR to fix it on release. Please take a look: https://github.com/wireapp/wire-ios/pull/1569/files

**Review by typfel — APPROVED** [review]

**Pull request merged**